docs/videos/dev.md

```diff
@@ -0,0 +1,13 @@
+# Overview
+
+The All Core Devs meeting is a public meeting of Aragon core developers. All developers working on all layers of the Aragon stack are welcome to join if they have an agenda item they'd like to discuss. The meetings are recorded, published, and documented here on the wiki. You can find documentation for past meetings under the All Core Devs section of the Aragon Wiki.
+
+# Participating in the All Core Devs meeting
+To participate in the call, you only have to have an agenda item accepted by one of the call organizers. These are the steps for proposing an agenda item:
+
+- Submit a pull request that adds a bullet point to the agenda linked in the Next Meeting section below.
+- A meeting organizer will review your Pull Request no later than 24 hours before the next meeting.
+- If your pull request is accepted, then your agenda item has been accepted.
+- You can request to be invited to join the meeting if you want to discuss your agenda item with the devs. If the organizers agree they will coordinate with you to add you to the meeting call. Otherwise the meeting participants will do their best to discuss your agenda item in your absense.
```

Comment:
How do we share the meeting link with them? I assume they'll leave an email or etc. for contact?

Reply:
That's a great question. Ethereum All Core Devs agenda process just has people send an email to Hudson. Perhaps we could create a shared inbox for this with a primary owner and backup owner in case the primary is OOO. After they email us, we can confirm their attendance and send them whatever they need to get added to the call.

Reply:
Maybe we could link the request and the review together.

Step 1. Send an email requesting your item be added to the agenda
2. If it is accepted, your item will be added to the agenda and you will be sent instructions for joining the call

This could reduce the back and forth and number of moving parts and address your other comments here.

Reply:
I think for now a google doc is easier than a shared inbox, but we can move to that if the google doc gets out of hand from people commenting / spamming.

Reply:
My thought with the shared inbox is that it enables us to communicate with contributors without requiring them to a) have a google account to comment on the doc and b) post their email address in public

Reply:
Hmm, good point. You can comment anonymously on the doc, but then we lose the ability to communicate back.

Let's set up the shared inbox, and communicate that as the recommended method for people to add items to the agenda then.